tools/chocolateyUninstall.ps1

```diff
@@ -1,20 +1,12 @@
-﻿function Get-RabbitMQPath
-{
-    $regPath = "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\RabbitMQ"
-    if (Test-Path "HKLM:\SOFTWARE\Wow6432Node\") { $regPath = "HKLM:\SOFTWARE\Wow6432Node\Microsoft\Windows\CurrentVersion\Uninstall\RabbitMQ" }
-    $path = Split-Path -Parent (Get-ItemProperty $regPath "UninstallString").UninstallString
-    $version = (Get-ItemProperty $regPath "DisplayVersion").DisplayVersion
-    return "$path\rabbitmq_server-$version"
-}
+﻿if(!$PSScriptRoot){ $PSScriptRoot = Split-Path $MyInvocation.MyCommand.Path -Parent }
+. "$PSScriptRoot\ChocolateyHelpers.ps1"
 
 $rabbitPath = Get-RabbitMQPath
 
-start-process -wait "$rabbitPath\..\uninstall" "/S"
+Start-Process -Wait "$rabbitPath\..\uninstall" "/S"
 
 $erlangKey = "HKLM:\SOFTWARE\Ericsson\Erlang\ErlSrv\1.1\RabbitMQ"
-
-if (Test-Path $erlangKey)
-{
-    Get-Process | Where-Object {$_.ProcessName -like "*erl*"} | Stop-Process -Force
-    Remove-Item $erlangKey
+if (Test-Path $erlangKey) {
+  Get-Process | Where-Object {$_.ProcessName -like "*erl*"} | Stop-Process -Force
```

Comment:
I'm a bit bothered by this wildcard and I'm not certain why its here. It seems like the hanging registry value is a problem for the rabbitmq scripts and not the chocolatey package, but I left it anyway.